engine/gameobject/src/gameobject/gameobject.cpp

```diff
@@ -1831,34 +1839,8 @@ namespace dmGameObject
         instance->m_ToBeAdded = 0;
     }
 
-    static void DoDeleteInstance(Collection* collection, HInstance instance)
```

Comment:
I just moved a part of DoDeleteInstance related to re-parenting into the other method to be able to call it from two different places